exporters/variables-scss/src/content/generators/cssGenerator.ts

```diff
@@ -0,0 +1,89 @@
+import { DimensionToken, StringToken, Token, TokenGroup, TokenType } from '@supernovaio/sdk-exporters';
+import { CSSHelper } from '@supernovaio/export-helpers';
+import { toPlural } from '../helpers/stringHelper';
+import { tokenVariableName } from '../helpers/tokenHelper';
+import { handleExceptions } from '../helpers/exceptionHelper';
+
+export const tokenToCSSByType = (
+  token: Token,
+  mappedTokens: Map<string, Token>,
+  tokenGroups: Array<TokenGroup>,
+  withParent: boolean,
+): string | null => {
+  if (token.tokenType === TokenType.dimension) {
+    const dimensionToken = token as DimensionToken;
+    const name = tokenVariableName(dimensionToken, tokenGroups, withParent);
+    let value = dimensionToken.value?.measure;
+    value = handleExceptions(name, value);
+    const unit = CSSHelper.unitToCSS(dimensionToken.value?.unit);
+
+    return `$${name}: ${value}${unit} !default;`;
+  }
+
+  if (token.tokenType === TokenType.string) {
+    const stringToken = token as StringToken;
+    const name = tokenVariableName(stringToken, tokenGroups, withParent);
+    let value = stringToken.value.text;
+    value = handleExceptions(name, value);
+
+    return `$${name}: ${value} !default;`;
+  }
+
+  return null;
+};
+
+export const generateCssFromTokens = (
+  tokens: Token[],
+  mappedTokens: Map<string, Token>,
+  tokenGroups: Array<TokenGroup>,
+  hasParentPrefix: boolean,
+): string => {
+  return tokens
+    .map((token) => tokenToCSSByType(token, mappedTokens, tokenGroups, hasParentPrefix))
+    .filter(Boolean)
+    .join('\n');
+};
+
+const generateObjectContent = (tokens: Array<Token>, tokenGroups: Array<TokenGroup>, withParent: boolean): string => {
+  return tokens.reduce((result, token) => {
```

Comment:
And same here.

Reply:
https://github.com/lmc-eu/spirit-design-system/pull/1592#discussion_r1736050473

Reply:
Ok.